model/src/main/proto/example.proto

```diff
@@ -9,6 +9,10 @@ message UserAppHistory {
   bool already_opened_app = 1;
 }
 
+message OnboardingFlow {
+  bool already_on_boarded_app = 1;
```

Comment:
Rename this to `already_onboarded_app` to keep the `Onboard` word consistent everywhere.

Reply:
Its on-board so will update where its is different